build/generate-opts.py

```diff
@@ -151,6 +151,11 @@ def __init__(self, items, **defaults):
     'help': 'A document or string that specifies the index to use to support the query predicate.'
 })
 
+let_option = ('let', {
+    'type': 'document',
+    'help': 'A BSON document consisting of any number of parameter names, each followed by definitions of constants in the MQL Aggregate Expression language.'
+})
```

Comment:
This covers all API methods and docs discussed in [DRIVERS-1788](https://jira.mongodb.org/browse/DRIVERS-1788) apart from `mongoc_collection_find_with_opts`. Since `find_with_opts` doesn't use the opts parser, I assume I'll need to manually add a line for the `let` option to the table in `mongoc_collection_find_with_opts.rst`, correct?

Reply:
I believe that is correct.